### DISCUSSION TITLE
How to use apply_ufunc depending on data dims?

### DISCUSSION THREAD

**Discussion by zxdawn** [pydata/xarray#6234] [Q&A] [answered]
Hi, I have posted one question on [Stack Overflow](https://stackoverflow.com/q/70910004/7347925), but no one answered yet. Briefly, that's how to apply our own function which depends on the dimension values? If anyone has an idea, please feel free to comment.

**Comment by dcherian** [comment, marked as answer]
I answered on stack overflow. Can you add a new section to https://xarray.pydata.org/en/stable/examples/apply_ufunc_vectorize_1d.html#Vectorization-with-np.vectorize showing this example? It seems like a good example to add.

**Reply by zxdawn** [reply]
Thanks for the nice answer! I will create a PR tomorrow.
